### PR TITLE
8 add service info

### PR DESCRIPTION
**Pull request by saupchurch**
Adds the /service-info endpoint see #8

### CODE REVIEW

File: rnaget-openapi.yaml [hunk 1]
```diff
@@ -1240,6 +1253,10 @@ components:
         message:
           type: string
           description: Error message details
+
```
Comment (emi80) [inline rnaget-openapi.yaml]: I would remove this blank line

Comment (saupchurch) [reply rnaget-openapi.yaml]: blank line removed and references to version incremented to 1.1.0

**Comment by saupchurch** [comment]
You have a good point - should think on this for a bit to see if there is anything we want to specify at a service level.  I'm not 100% on consolidating the `formats` and `filters` returns as the objects start to bleed into `/service-info` and it raises the question of what happens if a service doesn't implement an endpoint that has properties in `/service-info`.

**Comment by emi80** [comment]
Interesting point. 
On one side I am also not sure about consolidating `formats` and `filters` into `/service-info`. On the other, now that `/service-info` has been officially approved, we could use it as a mean to provide the `formats` and `filters` information and remove all the corresponding endpoints for a cleaner API structure. 
Though, I expect this to be seen as a breaking change...

**Comment by jb-adams** [comment]
@saupchurch @emi80 

Just summarizing what was discussed on today's RNAseq call:
- The group decided not to introduce any breaking changes, or changes that would break the encapsulation of class-specific data (supported filters and formats) being associated with their API routes
- Instead, `service-info` could support a basic true/false for each set of API routes, which will inform the client of whether those routes have been implemented or not

This could be achieved by an **optional** `supported` object property, which has 4 **optional** boolean properties, one for each API. For example:
```
{
    "supported": {
        "projects": true,
        "studies": true,
        "expressions": true,
        "continuous": false
    }
}
```

If any boolean property is not provided, it is assumed to be implemented. If the entire `supported` object is not provided, all APIs are assumed to be implemented.

Thoughts?

**Comment by saupchurch** [comment]
I added the Supported object as described along with a usage description.  Let me know what you think.

**Comment by emi80** [comment]
Hi all, I am sorry I was not very active in the call yesterday but I had to join from a shared space and could not talk much.

The PR looks good to me.

**Pull request merged**